pkg/util/lifted/pubkeypin/pubkeypin_test.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 // testCertPEM is a simple self-signed test certificate issued with the openssl CLI:
-// openssl req -new -newkey rsa:2048 -days 36500 -nodes -x509 -keyout /dev/null -out test.crt
+// openssl req -new -newkey rsa:3072 -days 36500 -nodes -x509 -keyout /dev/null -out test.crt
```

Comment:
Do the test certificate also need to be regenerated? The rsa key size of testCertPEM is 2048
```bash
(base) ➜  karmada git:(main) ✗  openssl x509 -noout -text -in test.crt
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number:
            98:15:68:25:ef:0b:a1:e4
        Signature Algorithm: sha1WithRSAEncryption
        Issuer: CN = kubeadm-keypins-test
        Validity
            Not Before: Jul  5 17:43:10 2017 GMT
            Not After : Jun 11 17:43:10 2117 GMT
        Subject: CN = kubeadm-keypins-test
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                RSA Public-Key: (2048 bit)
```

Reply:
nice, done